packages/stark-ui/assets/styles/_base.scss

```diff
@@ -17,15 +17,29 @@ body {
     margin-bottom: 64px;
   }
 
-  main.stark-main-container {
-    flex: 1 0 auto;
-    align-self: center;
+  .mat-drawer-content {
+    overflow: visible;
```

Comment:
@christophercr you are right, this line makes Firefox not respect the `height: 100%` of `.mat-drawer-content` (making it impossible to scroll in the `.stark-scroll-container`). The problem is... if I put set this to `overflow: hidden` Safari does not show the `<header>`. Do you or @nicanac or @SuperITMan  have any suggestion on the best way to resolve this? 🤔